modules/ocf_kubernetes/manifests/worker.pp

```diff
@@ -7,6 +7,8 @@
 # with the small requirement that kubernetes::worker
 # class is added.
 class ocf_kubernetes::worker {
+  package { 'nfs-common': }
```

Comment:
Are you sure this is necessary? Isn't all the NFS stuff running in Docker anyways?

Reply:
Yes, Kubernetes is mounting the NFS stuff on the workers, it's not being done in Docker.

Reply:
mounting NFS inside the container would require advanced privileges IIRC and is recommended against